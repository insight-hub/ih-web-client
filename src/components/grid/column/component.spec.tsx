import React from 'react';
import renderer from 'react-test-renderer';

import { Column } from './';

describe('Column', () => {
  it('should render correct', () => {
    const component = renderer.create(<Column cols={1} />);
    expect(component.toJSON()).toMatchSnapshot();
  });

  it('should render with breakpoints', () => {
    const component = renderer.create(
      <Column
        cols={{ xs: 2, sm: 5, md: 2, lg: 2, xl: 1 }}
        offset={{ xs: 2, sm: 5, md: 2, lg: 2, xl: 1 }}
      />,
    );
    expect(component.toJSON()).toMatchSnapshot();
  });
});
