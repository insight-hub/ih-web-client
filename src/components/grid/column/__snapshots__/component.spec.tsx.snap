// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Column should render correct 1`] = `
<div
  className="sc-bcXHqe ilWaIG"
  cols={1}
/>
`;

exports[`Column should render with breakpoints 1`] = `
<div
  className="sc-bcXHqe ipUaYV"
  cols={
    {
      "lg": 2,
      "md": 2,
      "sm": 5,
      "xl": 1,
      "xs": 2,
    }
  }
  offset={
    {
      "lg": 2,
      "md": 2,
      "sm": 5,
      "xl": 1,
      "xs": 2,
    }
  }
/>
`;
